serial_server.py

```diff
@@ -93,10 +130,13 @@ def list_serial_ports(self, c):
         NOTES:
         This list contains all ports installed on the computer,
         including ones that are already in use by other programs."""
-        return self.SerialPorts
+        print self.SerialPorts
+        port_list = [x.name for x in self.SerialPorts]
+
+        return port_list
 
     @setting(10, 'Open',
-             port=[': Open the first available port',
+             port=[': Open the first available port', # Is this really useful??? - ERJ
```

Comment:
This is probably not useful, but let's have the discussion here or in a separate issue instead of inline comments. I would be fine to remove.